jax/random.py

```diff
@@ -383,6 +383,34 @@ def _normal(key, shape, dtype):
   return onp.array(onp.sqrt(2), dtype) * lax.erf_inv(u)
 
 
+def truncated_normal(key, lower, upper, shape=(), dtype=onp.float64):
+  """Sample truncated standard normal random values with given shape and float
```

Comment:
Could you shorten this to fit on one line? This is just a style suggestion, which you can ignore if you want, but we've maintained it elsewhere in the code afaik.